src/bin/update_snapshots/main.rs

```diff
@@ -12,16 +12,25 @@ mod test_case;
 
 use tests::snapshot_tests;
 
+use crate::utils::{find_unused_snapshots, snapshots_path};
+
 fn main() {
+    let mut produced_snapshots = HashSet::new();
```

Comment:
I would prefer if gathering those snapshots would not be a part of loop that runs tests. You could add a method to TestCase that returns paths for all tests that should be produced.